ironfish/src/node.ts

```diff
@@ -297,6 +304,7 @@ export class IronfishNode {
       privateIdentity,
       hostsStore,
       networkId: networkDefinition.id,
```

Comment:
We could put a default in the node, but this seems better so that we can know if people are accessing node via different mechanism than SDK.